src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -255,10 +256,21 @@ export default class TableComponent extends ComponentBase implements IComponent,
 			this._table.focus();
 		}
 
+		if (this.checked !== undefined) {
+			this.check(this.checked);
```

Comment:
@chlafreniere asked initially if it will be better to have table.checked = list<checkboxInfos> so save multiple calls. I'll do that in the branch in sometime you want to take that.

Reply:
pushed this to other branch

Reply:
Took the changes from the other branch